helm/charts/amplication/values-staging.yaml

```diff
@@ -150,4 +150,4 @@ amplication-storage-gateway:
       BUILD_ARTIFACTS_BASE_FOLDER: /amplication-data/build-data
       BUILD_ARTIFACTS_CODE_FOLDER: code
       GITHUB_APP_APP_ID: "174833"
-
+      
```

Comment:
@abrl91 have your run npx nx format:write? If so we might need to review the prettier rules here..

Reply:
I added it manually before Ariel fixed the formatting because on one of my previous commits I deleted the last empty line by mistake